SML/SML/ExtensionMethods/ArrayExtension.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace SML.ExtensionMethods;
+
+public static class ArrayExtension
+{
+    #region Fields
+
+    private static string To2DExceptionMessage =
+        "The given jagged array is not rectangular";
+
+    #endregion Fields
+
+    #region Methods
+
+    public static IEnumerable<T> GetColumn<T>(this T[,] array, int column)
+    {
+        for (var i = 0; i < array.GetLength(0); i++)
```

Comment:
1) About "var": We use var in places, where it's clear what type it is ([link](https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#implicitly-typed-local-variables))
2) About "index": I think it's redundant, because "i" is more familiar for developers and it has similar sense with "index"